package.json

```diff
@@ -161,6 +161,7 @@
     "nodemon": "^2.0.4",
     "ora": "^4.0.4",
     "parse-prometheus-text-format": "^1.1.1",
+    "prettier": "^2.0.5",
```

Comment:
do we want to add prettier to one-app ?

Reply:
This is a peerDep required for inline snapshots, thats all I added it for